### ISSUE TITLE
a11y bug - duplicate aria-labelledby attributes on home page sections

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#9415]
**Describe the bug**
Axe Core has flagged an issue where we have duplicate `aria-labelledby` attributes on home page sections. This violates the `landmark-unique` rule in Axe Core - https://dequeuniversity.com/rules/axe/4.1/landmark-unique

<img width="1661" alt="Screenshot 2021-09-10 at 11 47 38" src="https://user-images.githubusercontent.com/4798332/132843405-0574ca06-70de-48b5-ae5d-1d66480c5682.png">

This is an example on the Arabic home page.

<img width="1155" alt="Screenshot 2021-09-10 at 11 49 21" src="https://user-images.githubusercontent.com/4798332/132843083-23aeccdb-3919-468c-9672-ddac1c66c18d.png">

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/arabic
2. Inspect a section element
3. Observe duplicated `aria-labelledby="Live-event-1"` attribute on the section element and on the link element

**Expected behaviour**
There should not be an `aria-labelledby` on the link element. This will fix the `landmark-unique` violation.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by mukama** [comment]
Resolved in https://github.com/bbc/psammead/pull/4553
